scrapper/models.go

```diff
@@ -126,23 +126,24 @@ func (r TableRow) TableFqn() DwhFqn {
 }
 
 type SqlDefinitionRow struct {
-	Instance string `db:"instance" json:"instance" ch:"instance" bigquery:"instance"`
-	Database string `db:"database" json:"database" ch:"_database" bigquery:"database"`
-	Schema   string `db:"schema" json:"schema" ch:"schema" bigquery:"schema"`
-	Table    string `db:"table" json:"table" ch:"table" bigquery:"table"`
-	IsView   bool   `db:"is_view" json:"is_view" ch:"is_view" bigquery:"is_view"`
-	Sql      string `db:"sql" json:"sql" ch:"sql" bigquery:"sql"`
+	Instance           string `db:"instance"             json:"instance"             ch:"instance"             bigquery:"instance"`
+	Database           string `db:"database"             json:"database"             ch:"_database"            bigquery:"database"`
+	Schema             string `db:"schema"               json:"schema"               ch:"schema"               bigquery:"schema"`
+	Table              string `db:"table"                json:"table"                ch:"table"                bigquery:"table"`
+	IsView             bool   `db:"is_view"              json:"is_view"              ch:"is_view"              bigquery:"is_view"`
+	IsMaterializedView bool   `db:"is_materialized_view" json:"is_materialized_view" ch:"is_materialized_view" bigquery:"is_materialized_view"`
```

Comment:
Added `IsMaterializedView` it is useful now for Trino scrapper to execute correct query to get DDL and can be useful in future for others scrappers.